rules/windows/process_creation/win_susp_diskshadow.yml

```diff
@@ -0,0 +1,26 @@
+title: Execution via Diskshadow.exe 
+id: 0c2f8629-7129-4a8a-9897-7e0768f13ff2
+status: experimental
+description: Detects using Diskshadow.exe to execute arbitrary code in text file
+references:
+    - https://bohops.com/2018/03/26/diskshadow-the-return-of-vss-evasion-persistence-and-active-directory-database-extraction/
+tags:
+    - attack.execution
+    - attack.t1218
+author: Ivan Dyachkov, oscd.community
+date: 2020/10/07  
+logsource:
+    category: process_creation 
+    product: windows            
+    definition: 'Requirements: Sysmon ProcessCreation logging must be activated and Windows audit msut Include command line in process creation events'
+detection:
+    selection:
+            Image: 'c:\windows\system32\diskshadow.exe'
```

Comment:
I think that it is better to look here only at the end of this field (image name) using the `endswith` modifier.